modules/installation-aws-about-government-region.adoc

```diff
@@ -41,6 +41,9 @@ ifdef::aws-secret[]
 The following AWS Secret Region partition is supported:
 
 * `us-iso-east-1`
+
+The MTU of the AWS SDN in the us-iso-east-1 region is not the same as AWS commercial. For configuring the MTU during cluster install, see _Installing a cluster on AWS with network customizations_.
```

Comment:
some comments:
1. Both SDN and OVN network types are applicable.
2. suggest s/us-iso-east-1/AWS Top Secret Region/, use common name instead of region name, like the name in  [1]
3. I thought it should be a note or an important box, and put it after section [2]


[1] https://docs.openshift.com/container-platform/4.9/installing/installing_aws/installing-aws-government-region.html#installation-aws-about-government-region_installing-aws-government-region
[2] https://docs.openshift.com/container-platform/4.9/installing/installing_aws/installing-aws-government-region.html#installation-configuration-parameters_installing-aws-government-region

Reply:
@Wiharris  thanks for the updates

> I incorporated the above changes and submitted a new PR. For the 3rd section in the link you provided, I noticed there were already two Notes and Important boxes right next to each other. To avoid a third box in the same location, I added the text in between the two boxes.

I'm thinking that since it's not a SDN or OVN-related parameter, it's focusing on the whole network, so can we make it more general? like `The maximum supported MTU in AWS Top Secret Region is not the same as AWS commercial ...`

> Also, I wanted to get more clarity on your second comment. For the s/us-iso-east-1/AWS Top Secret Region/, I could not find anywhere in the document that uses that name. Could you point to where in the documentation the common name is used instead of the region name?

[OSDOCS-2996](https://issues.redhat.com/browse/OSDOCS-2996) Secret region section actually uses Top Secret code (https://github.com/openshift/openshift-docs/pull/40350)

There are two regions in C2S, `us-iso-east-1` and `us-iso-west-1`, in the official AWS C2S Document, the MTU content is under `How Amazon EC2 Differs for C2S` section:
> How Amazon EC2 Differs for C2S
> ...
> For Windows instances, the maximum transmission unit (MTU) of a network connection is 1500 bytes. For Linux instances, the maximum supported MTU is 8868 bytes instead of 9001 bytes. However, the path MTU has been reduced to 8801 bytes in the region to enable Enhanced Networking on instances that support them. For more information, see Network Maximum Transmission Unit (MTU) for Your EC2 Instance.
> ...

Since we referred to the official document,  so I think the name (`AWS Top Secret Region`)  is better than the region name (`us-iso-east-1` and `us-iso-west-1`)

And another comment is, is it possible that add a web link for  _Installing a cluster on AWS with network customizations_?

Reply:
Thank you for clarifying! I have incorporated your suggestions. I was able to place the link in the Additional Resources section right after the passage appears.

Reply:
The content LGTM, but it looks like we need to apply this change to `installing-aws-secret-region` page [1] instead of `installing-aws-government-region` page (need to rebase to apply [2] and [3] changes?)

It’s a little confusing here, before [2] and [3] merged, GovCloud and secret regions are in the same doc [4], after that, they were separated into different pages.

cc @mjpytlak 

[1] https://deploy-preview-39769--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-secret-region
[2] https://github.com/openshift/openshift-docs/pull/39769
[3] https://github.com/openshift/openshift-docs/pull/39614
[4] https://docs.openshift.com/container-platform/4.9/installing/installing_aws/installing-aws-government-region.html